.codecov.yml

```diff
@@ -7,10 +7,12 @@ coverage:
         threshold: 0%
         if_no_uploads: failure
         if_ci_failed: failure
-    patch:off
+    patch: off
     changes:
       default:
+        set_pending: no
         if_no_uploads: failure
         if_ci_failed: failure
+        only_pulls: no
```

Comment:
I really need to open a review account, disable admin force merging, and start reviewing my own stuff. I almost let this in.
Should be yes.